ironfish-cli/src/commands/start.ts

```diff
@@ -90,45 +95,46 @@ export default class Start extends IronfishCommand {
     this.startDonePromise = startDonePromise
 
     const { flags } = this.parse(Start)
+    const { bootstrap, forceMining, graffiti, listen, name, port, worker, workers } = flags
 
-    if (flags.bootstrap !== undefined) {
-      this.sdk.config.setOverride('bootstrapNodes', flags.bootstrap.filter(Boolean))
+    if (bootstrap !== undefined) {
+      this.sdk.config.setOverride('bootstrapNodes', bootstrap.filter(Boolean))
+    }
+    if (port !== undefined && port !== this.sdk.config.get('peerPort')) {
+      this.sdk.config.setOverride('peerPort', port)
     }
-    if (flags.port !== undefined && flags.port !== this.sdk.config.get('peerPort')) {
-      this.sdk.config.setOverride('peerPort', flags.port)
+    if (workers !== undefined && workers !== this.sdk.config.get('nodeWorkers')) {
+      this.sdk.config.setOverride('nodeWorkers', workers)
     }
-    if (flags.workers !== undefined && flags.workers !== this.sdk.config.get('nodeWorkers')) {
-      this.sdk.config.setOverride('nodeWorkers', flags.workers)
+    if (name !== undefined && name.trim() !== this.sdk.config.get('nodeName')) {
+      this.sdk.config.setOverride('nodeName', name.trim())
     }
-    if (flags.name !== undefined && flags.name.trim() !== this.sdk.config.get('nodeName')) {
-      this.sdk.config.setOverride('nodeName', flags.name.trim())
+    if (listen !== undefined && listen !== this.sdk.config.get('enableListenP2P')) {
+      this.sdk.config.setOverride('enableListenP2P', listen)
     }
-    if (flags.listen !== undefined && flags.listen !== this.sdk.config.get('enableListenP2P')) {
-      this.sdk.config.setOverride('enableListenP2P', flags.listen)
+    if (worker !== undefined && worker !== this.sdk.config.get('isWorker')) {
+      this.sdk.config.setOverride('isWorker', worker)
     }
-    if (flags.worker !== undefined && flags.worker !== this.sdk.config.get('isWorker')) {
-      this.sdk.config.setOverride('isWorker', flags.worker)
+    if (forceMining !== undefined && forceMining !== this.sdk.config.get('miningForce')) {
+      this.sdk.config.setOverride('miningForce', forceMining)
     }
-    if (
-      flags.forceMining !== undefined &&
-      flags.forceMining !== this.sdk.config.get('miningForce')
-    ) {
-      this.sdk.config.setOverride('miningForce', flags.forceMining)
+    if (graffiti !== undefined && graffiti) {
```

Comment:
minor question, but now this is the only one that still sets override even if it's the same so the behaviour is slightly different.

Reply:
Woops, totally forgot to finish the rest of the check here. Updated and added a test.